packages/arb-token-bridge-ui/src/hooks/CCTP/useApproveAndDeposit.ts

```diff
@@ -49,26 +47,36 @@ export function useApproveAndDeposit({
         spender: tokenMessengerContractAddress
       })
 
-      if (allowance.lte(amountRaw)) {
-        const shouldContinue = await onAllowanceTooLow?.()
+      if (allowance.lt(amountRaw)) {
+        const shouldContinue = onAllowanceTooLow
+          ? await onAllowanceTooLow()
+          : true
         if (!shouldContinue) {
           return
         }
```

Comment:
this looks funky, maybe don't use the `shouldContinue` var and put the condition there directly?